ui/lib/components/ColumnsSelector/index.tsx

```diff
@@ -119,19 +119,28 @@ export default function ColumnsSelector({
   )
 
   const content = (
-    <Space direction="vertical">
-      {filteredColumns.map((column) => (
-        <Checkbox
-          key={column.key}
-          checked={visibleKeys[column.key]}
-          onChange={(e) => handleCheckChange(e, column)}
-        >
-          {column.name}
-        </Checkbox>
-      ))}
-
+    <div style={{ marginTop: -12 }}>
```

Comment:
Why do we need this `marginTop: -12`?

Reply:
The effect with default margin:

![image](https://user-images.githubusercontent.com/1284531/94355420-7d3b0580-00b6-11eb-9b27-ec513291662d.png)